src/engraving/dom/score.cpp

```diff
@@ -1131,7 +1131,7 @@ std::vector<System*> Score::searchSystem(const PointF& pos, const System* prefer
         }
         if (y < y2) {
             systems.push_back(s);
-            for (size_t iii = i + 1; ii < n; ++iii) {
+            for (size_t iii = i + 1; iii < n; ++iii) {
```

Comment:
Even without thinking through the code, this much seems like a straight up bug fix, right?

Reply:
Yes, but I don't think it has any effect because the `break` below would be hit.